domains/error.go

```diff
@@ -31,6 +31,12 @@ func (e AssertError) Error() string {
 // ErrorCode identifies a kind of error.
 type ErrorCode int
 
+// These constants are used to identify errors specific to merkleroots
+const (
+	MerklerootNotFoundError          = "No block with provided merkleroot was found"
+	MerklerootNotInLongestChainError = "Provided merkleroot is not part of the longest chain"
+)
```

Comment:
```suggestion
var (
	MerklerootNotFoundError          = errors.New("No block with provided merkleroot was found")
	MerklerootNotInLongestChainError = errors.New("Provided merkleroot is not part of the longest chain")
)
```

Consider this 😄 It will require more changes in places where the errors are actually returned.

Reply:
Allright will change that